### ISSUE TITLE
Reproduced test accuracy on finetuned checkpoint is 1% inferior on Kinetics-400

### ISSUE THREAD

**Issue by anirudth** [MCG-NJU/VideoMAE#65]
Hi, 

Thanks for the amazing work! I want to reproduce your test accuracy with your finetuned checkpoint for my experiment baselines. 

I downloaded the finetuned checkpoint of ViT-S and run it on Kinetics-400 dataset. The top-1 accuracy I obtain is 78% versus 79% that's reported in the repo. 

The following are the commands I use to run and corresponding log:

```
# Set the path to save checkpoints
OUTPUT_DIR='logs/k400_videomae_pretrain_small_patch16_224_frame_16x4_tube_mask_ratio_0.9_e1600/eval_lr_5e-4_ep_150'
# path to Kinetics set (train.csv/val.csv/test.csv)
DATA_PATH='dataset/VideoMAE/kinetics-dataset/k400/videos'
# path to pretrain model
MODEL_PATH='models/vits_kinetics_finetuned_checkpoint.pth'

# We add repeated_aug (--num_sample = 2) on Kinetics-400 here, 
# which could better performance while need more time for fine-tuning

# batch_size can be adjusted according to number of GPUs
# this script is for 32 GPUs (4 nodes x 8 GPUs)
OMP_NUM_THREADS=1 python -m torch.distributed.launch --nproc_per_node=8 \
    --master_port 12320 --nnodes=1  --node_rank=$1 --master_addr=$2 \
    run_class_finetuning.py \
    --model vit_small_patch16_224 \
    --data_set Kinetics-400 \
    --nb_classes 400 \
    --data_path ${DATA_PATH} \
    --finetune ${MODEL_PATH} \
    --log_dir ${OUTPUT_DIR} \
    --output_dir ${OUTPUT_DIR} \
    --batch_size 72 \
    --num_sample 2 \
    --input_size 224 \
    --short_side_size 224 \
    --save_ckpt_freq 10 \
    --num_frames 16 \
    --sampling_rate 4 \
    --opt adamw \
    --lr 5e-4 \
    --opt_betas 0.9 0.999 \
    --weight_decay 0.05 \
    --epochs 150 \
    --test_num_segment 5 \
    --test_num_crop 3 \
    --dist_eval \
    --enable_deepspeed --eval
```

The following are the logs on my terminal

```
  warnings.warn("video {}, temporal {}, spatial {} not found during testing".format(\
Test:  [ 970/1008]  eta: 0:04:47  loss: 1.2984 (1.3929)  acc1: 69.4444 (69.4502)  acc5: 87.5000 (87.5343)  time: 6.2067  data: 5.6619  max mem: 7782
/home/ubuntu/workplace/VideoMAE/VideoMAE/kinetics.py:130: UserWarning: video /mnt/efs/fs1/data/personal/anirudtn/VideoMAE/kinetics-dataset/k400/test/Lw14NH9kAqE_000759_000769.mp4, temporal 4, spatial 2 not found during testing
  warnings.warn("video {}, temporal {}, spatial {} not found during testing".format(\
/home/ubuntu/workplace/VideoMAE/VideoMAE/kinetics.py:130: UserWarning: video /mnt/efs/fs1/data/personal/anirudtn/VideoMAE/kinetics-dataset/k400/test/2xWiEVNUvhE_000064_000074.mp4, temporal 4, spatial 2 not found during testing
  warnings.warn("video {}, temporal {}, spatial {} not found during testing".format(\
/home/ubuntu/workplace/VideoMAE/VideoMAE/kinetics.py:130: UserWarning: video /mnt/efs/fs1/data/personal/anirudtn/VideoMAE/kinetics-dataset/k400/test/jJFqy6yiXzQ_000024_000034.mp4, temporal 4, spatial 2 not found during testing
  warnings.warn("video {}, temporal {}, spatial {} not found during testing".format(\
/home/ubuntu/workplace/VideoMAE/VideoMAE/kinetics.py:130: UserWarning: video /mnt/efs/fs1/data/personal/anirudtn/VideoMAE/kinetics-dataset/k400/test/CxjipYE57Yo_000199_000209.mp4, temporal 4, spatial 2 not found during testing
  warnings.warn("video {}, temporal {}, spatial {} not found during testing".format(\
Test:  [ 980/1008]  eta: 0:03:31  loss: 1.5340 (1.3965)  acc1: 66.6667 (69.3666)  acc5: 86.1111 (87.4972)  time: 6.0654  data: 5.5356  max mem: 7782
/home/ubuntu/workplace/VideoMAE/VideoMAE/kinetics.py:130: UserWarning: video /mnt/efs/fs1/data/personal/anirudtn/VideoMAE/kinetics-dataset/k400/test/y7cYaYX4gdw_000047_000057.mp4, temporal 4, spatial 2 not found during testing
  warnings.warn("video {}, temporal {}, spatial {} not found during testing".format(\
Test:  [ 990/1008]  eta: 0:02:15  loss: 1.6756 (1.3990)  acc1: 62.5000 (69.3169)  acc5: 83.3333 (87.4566)  time: 5.5729  data: 5.0751  max mem: 7782
Test:  [1000/1008]  eta: 0:01:00  loss: 1.6163 (1.4011)  acc1: 63.8889 (69.2627)  acc5: 86.1111 (87.4334)  time: 5.2124  data: 4.7285  max mem: 7782
Test:  [1007/1008]  eta: 0:00:07  loss: 1.6333 (1.4022)  acc1: 66.6667 (69.2562)  acc5: 83.3333 (87.3978)  time: 4.7048  data: 4.2354  max mem: 7782
Test: Total time: 2:05:55 (7.4954 s / it)
* Acc@1 69.153 Acc@5 87.293 loss 1.408
Start merging results...
Reading individual output files
Computing final results
38671
Accuracy of the network on the 580275 test videos: Top-1: 78.00%, Top-5: 93.16%
```

Where do you think the gap is coming from?

**Comment by yztongzhan** [comment]
Hi @anirudth! Thanks for your interest in our work!
The short side of each video in validation set is 320. It might result in lower performance if you resize videos or extract frames with lower resoultion.
Moreover, MMAction2 has successfully reproduced the results with our checkpoints on Kinetics-400. Maybe you can get help from their [implementation](https://github.com/open-mmlab/mmaction2/tree/dev-1.x/configs/recognition/videomae#results-and-models).

**Comment by potatowarriors** [comment]
The short side in the published config is set to 256, does this mean that both K400 and SSV2 need a short_side_size of 320 to reproduce? @yztongzhan
